src/main/java/org/wildfly/security/util/ServiceLoaderSupplier.java

```diff
@@ -65,7 +65,7 @@ public ServiceLoaderSupplier(final Class<E> service, final ClassLoader classLoad
         return result.clone();
     }
 
-    private E[] loadServices(final Class<E> service, final ClassLoader classLoader) {
+    protected E[] loadServices(final Class<E> service, final ClassLoader classLoader) {
```

Comment:
I think we should not make this ``protected``; it would needlessly increase our API surface area (and there's no guarantee we wouldn't want to refactor at this level).  Package-private is probably the better choice here.